pom.xml

```diff
@@ -0,0 +1,499 @@
+        <module>servicebus/spring-cloud-azure-starter-servicebus-jms/servicebus-jms-topic</module>
+        <module>servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-queue-binder</module>
+        <module>servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-multibinders</module>
+        <module>servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-topic-binder</module>
+        <module>servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-queue-binder-arm</module>
+        <module>servicebus/spring-cloud-azure-starter-integration-servicebus/multiple-namespaces</module>
+        <module>servicebus/spring-cloud-azure-starter-integration-servicebus/single-namespace</module>
+        <module>storage/spring-cloud-azure-starter-storage-blob/storage-blob-sample</module>
+        <module>storage/spring-cloud-azure-starter-storage-file-share/storage-file-sample</module>
+        <module>storage/spring-cloud-azure-starter-integration-storage-queue/storage-queue-operation</module>
+        <module>storage/spring-cloud-azure-starter-integration-storage-queue/storage-queue-integration</module>
+        <module>storage/spring-cloud-azure-starter-storage-queue/storage-queue-client</module>
+      </modules>
+      <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <java.version>17</java.version>
+        <version.spring.boot>3.0.0-RC2</version.spring.boot>
+        <version.spring.cloud>2022.0.0-RC2</version.spring.cloud>
+        <version.spring.cloud.azure>6.0.0-beta.4</version.spring.cloud.azure>
+        <version.thymeleaf-extras-springsecurity6>3.1.1.RELEASE</version.thymeleaf-extras-springsecurity6>
+        <version.maven-surefire-plugin>2.22.0</version.maven-surefire-plugin>
+        <mockito.version>4.8.1</mockito.version>
+      </properties>
+      <dependencyManagement>
+        <dependencies>
+          <dependency>
+            <groupId>org.thymeleaf.extras</groupId>
+            <artifactId>thymeleaf-extras-springsecurity6</artifactId>
+            <version>${version.thymeleaf-extras-springsecurity6}</version>
+          </dependency>
+        </dependencies>
+      </dependencyManagement>
+      <repositories>
+        <repository>
+          <id>spring-milestones</id>
+          <name>Spring Milestones</name>
+          <url>https://repo.spring.io/milestone</url>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </repository>
+      </repositories>
+      <pluginRepositories>
+        <pluginRepository>
+          <id>spring-milestones</id>
+          <name>Spring Milestones</name>
+          <url>https://repo.spring.io/milestone</url>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </pluginRepository>
+      </pluginRepositories>
+    </profile>
+    <profile>
+      <id>buildpack-4.x</id>
```

Comment:
If `-4.x` means `spring-cloud-azure-4.x`, how about renaming to `build-pack-for-spring-cloud-azure-4.x`?

Reply:
It may be too long, it can not use standalone, it should combine the `spring-cloud-azure-4.x` and `buildpack-4.x`.